packages/perseus-editor/src/issues-panel.tsx

```diff
@@ -20,10 +23,18 @@ export type Issue = {
 };
 
 type IssuesPanelProps = {
+    apiOptions?: APIOptions;
```

Comment:
do you think we would need this in the future? currently we are only using it for the feature flag.
```suggestion
    // TODO(LEMS-3520)
    apiOptions?: APIOptions;
```